src/components/TraceGesture.tsx

```diff
@@ -65,23 +65,36 @@ const TraceGesture = ({ eventNodeRef }: TraceGestureProps) => {
     const signaturePad = signaturePadRef.current['signaturePad']
     const eventNode = eventNodeRef?.current
 
-    // Attach pointer handlers to a provided node rather than the signature pad canvas.
+    // Attach touch handlers to a provided node rather than the signature pad canvas.
     // See: eventNodeRef
-    const handlePointerStart = signaturePad._handlePointerStart.bind(signaturePad)
-    const handlePointerMove = signaturePad._handlePointerMove.bind(signaturePad)
+    const handleTouchStart = signaturePad._handleTouchStart.bind(signaturePad)
+    const handleTouchMove = signaturePad._handleTouchMove.bind(signaturePad)
+    const handleTouchEnd = signaturePad._handleTouchEnd.bind(signaturePad)
 
-    eventNode?.addEventListener('pointerdown', e => {
+    eventNode?.addEventListener('touchstart', e => {
       // Make preventDefault a noop otherwise tap-to-edit is broken.
       // e.cancelable is readonly and monkeypatching preventDefault is easier than copying e.
       e.preventDefault = noop
-      const shouldActivateGesture = isInGestureZone(e.clientX, e.clientY, leftHanded)
-      if (shouldActivateGesture) {
-        handlePointerStart(e)
+
+      const touch = e.touches[0]
+      if (isInGestureZone(touch.clientX, touch.clientY, leftHanded)) {
+        handleTouchStart(e)
+      }
+    })
+
+    eventNode?.addEventListener('touchmove', e => {
+      const touch = e.touches[0]
+      if (isInGestureZone(touch.clientX, touch.clientY, leftHanded)) {
+        handleTouchMove(e)
```

Comment:
we should check for this too, as blindly monkey-patching `event.preventDefault` stops our scroll prevention

but this way we can just trigger the trace only on our gesture side, so it works very well